stack.py

```diff
@@ -9,10 +9,10 @@ def push(self, val):
         self.storage.push(val)
 
     def pop(self):
-        if self.storage:
+        try:
             return self.storage.pop()
-
-        raise ValueError("The stack is empty.")
+        except:
```

Comment:
This called a bare exception and generally should not be used.  Only except the exception(s) that you expect could happen.